src/plugins/vis_types/timeseries/public/application/components/lib/create_number_handler.ts

```diff
@@ -12,6 +12,9 @@ import { TimeseriesVisParams } from '../../../types';
 export const createNumberHandler = (
   handleChange: (partialModel: Partial<TimeseriesVisParams>) => void
 ) => {
-  return (name: keyof Metric, defaultValue?: string) => (e: React.ChangeEvent<HTMLInputElement>) =>
-    handleChange?.({ [name]: Number(e.target.value ?? defaultValue) });
+  return (name: keyof Metric, defaultValue?: string, isOptional?: boolean) =>
```

Comment:
not sure that it should be modified, I think we can do like in other places in TSVB. 
In case of "" we show a default value in the placeholder. 

in accordance to do it's  `1`

Reply:
added placeholder to input